File: app/src/main/java/org/exallium/tradetracker/app/model/rest/MtgPriceRestService.java
package org.exallium.tradetracker.app.model.rest;

import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

import java.util.List;

public interface MtgPriceRestService {
    @GET("/cfb/price.json")
    Observable<List<String>> getPriceInfo(@Query("cardname") String name, @Query("cardset") String set);
}

File: app/src/main/kotlin/org/exallium/tradetracker/app/model/rest/MtgPriceRestService.kt
package org.exallium.tradetracker.app.model.rest

import retrofit.http.GET
import retrofit.http.Query
import rx.Observable

public trait MtgPriceRestService {
    GET("/cfb/price.json")
    public fun getPriceInfo(Query("cardname") name: String, Query("cardset") set: String): Observable<List<String>>
}